tests/unit/models/continuity/ContETFStatic.spec.ts

```diff
@@ -0,0 +1,227 @@
+import StuntSheetDot from "@/models/StuntSheetDot";
+import ContETFStatic from "@/models/continuity/ContETFStatic";
+import { DIRECTIONS, MARCH_TYPES } from "@/models/util/constants";
+import { FlowBeat } from "@/models/util/types";
+import { CONT_IDS } from "@/models/continuity/BaseCont";
+
+describe("models/continuity/ETFStatic", () => {
+  const startDot = new StuntSheetDot({ x: 2, y: 4 });
+
+  it("has correct continuityId", () => {
+    const continuity = new ContETFStatic();
+    expect(continuity.continuityId).toBe(CONT_IDS.ETF_STATIC);
+  });
+
+  describe("getHumanReadableText", () => {
+    it("generates FMHS 8 E", () => {
```

Comment:
Can you create a test where facingDirection isn't defined, and make sure that the constructor makes it so `facingDirection` equals `marchingDirection`?

Reply:
Good idea!